code/modules/shuttle/ftl.dm

```diff
@@ -48,6 +48,46 @@
 	width = SSmapping.config.ftl_ship_width
 	height = SSmapping.config.ftl_ship_height
 
+/obj/docking_port/mobile/fob
+	name = "FTL FOB"
+	id = "fob"
+	callTime = 650
+	preferred_direction = SOUTH
+	area_type = /area/shuttle/ftl/cargo/mining
+	dir = NORTH
+	dwidth = 15
+	dheight = 0
+	width = 22
+	height = 13
```

Comment:
add new values to the json for the fob and make the docks use them so mappers can have different sizes

Reply:
Standard docking port practice is to var edit these values in a map editor. This is just a default since Aetherwhisp is the default map.

Reply:
you'll still need to add fob shuttle sizes to the json for maps which have different size shuttles

Reply:
bleh, fine